app/src/main/java/com/framgia/fsalon/screen/customerinfo/CustomerInfoViewModel.java

```diff
@@ -0,0 +1,65 @@
+package com.framgia.fsalon.screen.customerinfo;
+
+import android.support.annotation.StringDef;
+import android.support.v7.app.AppCompatActivity;
+
+import com.framgia.fsalon.R;
+import com.framgia.fsalon.screen.info.InfoUserFragment;
+
+/**
+ * Exposes the data to be used in the CustomerInfo screen.
+ */
+public class CustomerInfoViewModel implements CustomerInfoContract.ViewModel {
+    private static final String INFO = "Info";
+    private static final String BILL = "Bill";
+    private static final String PICTURE = "Picture";
+    private CustomerInfoContract.Presenter mPresenter;
+    private CustomerInfoAdapter mPagerAdapter;
+    private int[] mImageSource = {R.drawable.ic_user, R.drawable
+        .ic_list, R.drawable.ic_picture};
```

Comment:
ERROR: 'array initialization' child have incorrect indentation level 8, expected level should be one of the following: 12, 16, 38, 41.